Packs/FeedMitreAttackv2/ReleaseNotes/1_1_44.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+
+##### MITRE ATT&CK
+Fixed an issue where the "get_mitre_value_from_id" command did not ignore revoked indicators.
```

Comment:
Do you need to include updated Docker image?

Reply:
Someone just updated it the commit before :)